packages/fullstack/src/server/mod.rs

```diff
@@ -133,15 +134,15 @@ pub trait DioxusRouterExt<S> {
     ///     let addr = dioxus_cli_config::fullstack_address_or_localhost();
     ///     let router = axum::Router::new()
     ///         // Server side render the application, serve static assets, and register server functions
-    ///         .serve_static_assets()
+    ///         .serve_static_assets(std::path::Path::new("/assets"))
     ///         // Server render the application
     ///         // ...
     ///         .into_make_service();
     ///     let listener = tokio::net::TcpListener::bind(addr).await.unwrap();
     ///     axum::serve(listener, router).await.unwrap();
     /// }
     /// ```
-    fn serve_static_assets(self) -> Self
+    fn serve_static_assets(self, pubic_path: &Path) -> Self
```

Comment:
Modifying this signature is not semver compatible meaning we wouldn't be able to release this in version 0.6.2.

I think we can add this with a default impl but since  DioxusRouterExt is not sealed, any changes to its signature will need to follow semver rules.

https://doc.rust-lang.org/cargo/reference/semver.html#trait-item-signature